app/src/main/java/com/library/PatronManager.java

```diff
@@ -0,0 +1,46 @@
+package com.library;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PatronManager {
```

Comment:
We can define a contract for both the book and patron inventories which would include functions like getting, saving etc.